Framework/HistogramData/src/Rebin.cpp

```diff
@@ -40,6 +40,13 @@ Histogram rebinCounts(const Histogram &input, const BinEdges &binEdges) {
     auto owidth = xo_high - xo_low;
     auto nwidth = xn_high - xn_low;
 
+    if (xo_high < -1e100 || xo_low < -1e100) {
```

Comment:
In this case is either `xo_high` or `xo_low` equal `DBL_MIN`? If so is there are reason not to `if (xo_high == DBL_MIN || xo_low < DBL_MIN)`?